docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/modules-configuration.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Modules configuration
+description: Modules configuration extension scenario
+last_updated: Mar 13, 2023
+template: concept-topic-template
+related:
+- title: Plugins registration
+  link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/event-subscribers-registration.html
+- title: Event subscribers registration
+  link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/event-subscribers-registration.html
+- title: Modules configuration
+  link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/modules-configuration.html
+---
+
+## Introduction
+
+Manifests support changes in module configuration files.
+
+Manifests support all scalar types (bool, int, float, string), compound type array and special type null.
+
+Manifests do not support compound types object, callable, iterable and special type resource.
+
+Manifest only add the values to configuration files. Manifests DO NOT REMOVE the values from project configuration. 
+
+## 1.1. Basic scalar values as return
+
+Manifests fully support 4 scalar types:
```

Comment:
instead of "4 scalar types" a more generic phrase can be used, e.g. the next "all PHP data types" to make it similar to the text above.